lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -187,9 +188,9 @@ function dosomething_reportback_menu() {
       'file' => 'dosomething_reportback.admin.inc',
     );
   }
-  $items['rb/%reportback_file'] = array(
```

Comment:
Suggesting that we still keep the `rb/[fid]` menu path, but that the page callback will change to call `dosomething_reportback_view_entity` but pass the fid (instead of grabbing the fid from the query string). Bringing it up more as a possibility/convo vs anything to change now